### PR TITLE
add PoE documentation

### PR DESCRIPTION
**Pull request by KanjiMonster**
This is #70 with commits to address the comments on top of it.

### CODE REVIEW

File: platform_configuration/power_over_ethernet.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  -s   PoE status
+  -mV  PoE voltage
+  -mC  PoE current
+  -mT  PoE temperature
+  -mP  PoE power
+```
+
+### Examples
+
+To enable PoE on a port run the following command:
```
Comment (jklare) [inline platform_configuration/power_over_ethernet.md:44]:
```suggestion
To enable PoE on all ports run the following command:
```

File: platform_configuration/power_over_ethernet.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```
+:~$ poectl -e port2
```
Comment (jklare) [inline platform_configuration/power_over_ethernet.md:47]:
```suggestion
:~$ poectl -e
```

File: platform_configuration/power_over_ethernet.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+To disable PoE on a port:
```
Comment (jklare) [inline platform_configuration/power_over_ethernet.md:50]:
```suggestion
To disable PoE on a certain port:
```

**Review by jklare — COMMENTED** [review]
lgtm, some suggestions

File: platform_configuration/power_over_ethernet.md (hunk 2 above)
Comment (KanjiMonster) [reply platform_configuration/power_over_ethernet.md:47]: While this does work, the help usage says you should use `all`, so we should stick with that.

Comment (jklare) [reply platform_configuration/power_over_ethernet.md:47]:
```suggestion
:~$ poectl -e all
```

**Comment by jklare** [comment]
lgtm

**Pull request merged**